bokeh/server/protocol/messages/patch_doc.py

```diff
@@ -66,6 +66,11 @@ def should_suppress_on_change(self, event):
                 if event_json['kind'] == 'RootRemoved' and \
                    event_json['model']['id'] == event.model._id:
                     return True
+        elif isinstance(event, TitleChangedEvent):
+            for event_json in self.content['events']:
+                if event_json['kind'] == 'TitleChanged' and \
+                   event_json['title'] == event.title:
+                    return True
```

Comment:
Sounds good. I have the callbacks PR coming through too so it's actually even better :)